backend/routes/books.js

```diff
@@ -17,7 +17,7 @@ router.get('/recent/:sessionToken', async (req, res) => {
         let bookQuery = new Parse.Query("Books")
         bookQuery.equalTo("userId", user)
         bookQuery.descending("createdAt")
-        let books = await bookQuery.find({useMasterKey : true})
+        let books = await bookQuery.find()
```

Comment:
Please consider making one commit per thing changed. For example this commit could have been divided into one for the changes made to the recommendation code and another one for the changes made on the friends code